Source/Core/VideoBackends/Vulkan/VKGfx.cpp

```diff
@@ -277,13 +278,13 @@ void VKGfx::BindBackbuffer(const ClearColor& clear_color)
     else
     {
       ERROR_LOG_FMT(VIDEO, "Unknown present error {:#010X} {}, please report.",
-                    static_cast<u32>(res), VkResultToString(res));
+                    Common::ToUnderlying(res), VkResultToString(res));
```

Comment:
This one is *not* a restoration, but it is probably more correct -- `VkResult` is clearly signed if you look at its defined enum values.

Reply:
That code moved from VKRenderer.cpp, a file that no longer exists but was one of the modifications in the aforementioned commit.

Reply:
Ah you're right, I found it. Still definitely more correct with `Common::ToUnderlying()`.

Reply:
Interesting, I did not know this; https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkResult.html says:

> Commands in Vulkan return their status via return codes that are in one of two categories:
> * Successful completion codes are returned when a command needs to communicate success or status information. All successful completion codes are non-negative values.
> * Run time error codes are returned when a command needs to communicate a failure that could only be detected at runtime. All runtime error codes are negative values.

Reply:
though also, printing it as a hex value is probably wrong since the specification gives them in decimal (and e.g. `VK_ERROR_SURFACE_LOST_KHR = -1000000000` would be `-0x3b9aca00` which isn't nice).